lib/src/screen/dice/roll_widget.dart

```diff
@@ -0,0 +1,39 @@
+import '../../styles/dimensions.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class RollWidget extends StatelessWidget {
+  final Future<int> vf;
+
+  const RollWidget({
+    Key key,
+    this.vf,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: FutureBuilder<int>(
+        future: vf,
+        builder: (context, AsyncSnapshot<int> snapshot) {
+          switch (snapshot.connectionState) {
+            case ConnectionState.active:
+            case ConnectionState.waiting:
+              return CircularProgressIndicator();
+            case ConnectionState.done:
+              if (snapshot.hasError)
+                return Text('Error: ${snapshot.error}');
+              else if (snapshot.hasData) {
+                return Text(
+                  '${snapshot.data}',
+                  style: TextStyle(
+                      fontWeight: FontWeight.bold, fontSize: Dimension.rollWidgetTextFontSize),
```

Comment:
check for trailing commas, font size and font weight should be on different lines

Reply:
Done!